src/hypergol/cli/templates/serve_model.py.j2

```diff
@@ -0,0 +1,71 @@
+import json
+from typing import List
+import fire
+import uvicorn
+import tensorflow as tf
+from fastapi import FastAPI
+from hypergol.utils import create_pydantic_type
+from models.{{ modelName.asSnake }}_batch_processor import {{ modelName }}BatchProcessor
+from data_models.{{ inputClass.asSnake }} import {{ inputClass }}
+from data_models.{{ outputClass.asSnake }} import {{ outputClass }}
+
+TITLE = 'Serve {{ modelName }}'
+VERSION = '0.1'
+DESCRIPTION = 'FastApi wrapper on {{ modelName }}, see /docs for API details'
+USE_GPU = False
+THREADS = None
+MODEL_DIRECTORY = '<data directory>/<project>/<branch>/models/{{ modelName }}/<epoch>'
```

Comment:
can't data dir and project etc be rendered too? Or do you just want to leave this to the user?

Reply:
at the create_model part the data doesn't exist yet, that's why only the fixed part is generated (plus models)